### PR TITLE
SEO-178901-Maui-Expander-Interlinking

### PR DESCRIPTION
**Pull request by Edithabich**
|Title|Description|
|--|--|
|Task Category| .NET MAUI Expander|
|Content Task Link/Mail Screenshot| https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Asyncfusion.com&metrics=CLICKS%2CIMPRESSIONS%2CCTR&start_date=20240101&end_date=20240110&page=*%2Freact-tree-grid , https://dev.azure.com/Syncfusion-General/Content/_workitems/edit/764046/ |
|UX task| NA|
|Interlinking PR link| Interlinking -https://github.com/syncfusion-content/maui-docs/pull/1833|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/179464/|
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-Maui-Expander--Money-keywords.aspx|

 

Worked on 16 keywords in the .NET MAUI Expander and am keeping track of all keyword details in the link below:-
[https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan---.NET-MAUI-Expander.aspx?Mode=Edit]

### CODE REVIEW

File: MAUI/Expander/appearance.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Appearance in .NET MAUI Expander (SfExpander)
 
-The .NET MAUI SfExpander comes with built-in support for customizing the appearance of the header.
+The [.NET MAUI SfExpander](https://www.syncfusion.com/maui-controls/maui-expander) comes with built-in support for customizing the appearance of the header.
```
Comment (Sharly-Atieno) [inline MAUI/Expander/appearance.md:12]: Use below
[.NET MAUI Expander]

File: MAUI/Expander/overview.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Overview of Expander (SfExpander)
 
-The .NET MAUI Expander (SfExpander) is a layout control capable of loading any UI view in both header and content sections. By simply tapping the header, users can effortlessly expand or collapse the control.
+The [.NET MAUI Expander](https://www.syncfusion.com/maui-controls/maui-expander) is a layout control capable of loading any UI view in both header and content sections. By simply tapping the header, users can effortlessly expand or collapse the control.
```
Comment (Sharly-Atieno) [inline MAUI/Expander/overview.md:13]: Revert this

File: MAUI/Expander/appearance.md (hunk 1 above)
Comment (Edithabich) [reply MAUI/Expander/appearance.md:12]: resolved

File: MAUI/Expander/overview.md (hunk 2 above)
Comment (Edithabich) [reply MAUI/Expander/overview.md:13]: resolved

**Review by Sharly-Atieno — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**